nodestream/extractors/stores/aws/s3_extractor.py

```diff
@@ -14,30 +14,45 @@ def from_file_data(
         bucket: str,
         prefix: Optional[str] = None,
         object_format: Optional[str] = None,
+        archive_dir: str = None,
```

Comment:
technically this is optional

```suggestion
        archive_dir: Optional[str] = None,
```

Reply:
Done.  Please double check me